aptos-move/e2e-move-tests/src/aggregator.rs

```diff
@@ -7,8 +7,7 @@ use language_e2e_tests::account::Account;
 use std::path::PathBuf;
 
 pub fn initialize(path: PathBuf) -> (MoveHarness, Account) {
-    // TODO: when parallel execution supports deltas - change to `new_mainnet`.
-    let mut harness = MoveHarness::new_no_parallel();
+    let mut harness = MoveHarness::new_mainnet();
```

Comment:
Why are you using the new_mainnet version here?

Reply:
to enable parallel execution I think

Reply:
Why not MoveHarness::new()?

Reply:
new_mainnet is fine but it disables a lot of things such as core resources account so if the test needs core resources, it should use MoveHarness::new(). I'm fine either way

Reply:
Fixed